docs/amber_script.md

```diff
@@ -183,22 +183,24 @@ needed `DIMS`. If the `FORMAT` is missing it is defaulted to
 TODO(dsinclair): Sync the BufferTypes with the list of Vulkan Descriptor types.
 
 
-When adding a buffer binding, the `IDX` parameter is optional and will default
-to 0.
-
 ```
+  # Bind the buffer of the given |buffer_type| at the given descritor set
+  # and binding. The buffer will use a start index of 0.
```

Comment:
I was confused by the "start index" statement for a while.  But it's the starting array index when the binding is an arrayed resource.  See the dstArrayElement in https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#VkWriteDescriptorSet

So no change, this works.

Reply:
Actually, the *Vulkan* call lets you update multiple descriptors in a row, and in that API call the index parameter is the start index.
But in AmberScript this call is binding a single entry.  So it's just an *index* instead of a start index.